js/components/ConnectionBar.js

```diff
@@ -19,13 +19,24 @@ type OwnProps = {}
 
 type Props = ReduxMappedProps & ReduxProps & OwnProps
 
+const styles = StyleSheet.create({
+  container: {
+    display: 'flex',
+    justifyContent: 'center',
+    height: 30,
+    paddingLeft: 20
+  }
+})
+
 /**
  * Implements a component that show a message when there is no network connection
  */
 class ConnectionBar extends React.PureComponent<Props> {
-  render() {
+  render(): ?React$Element<*> {
```

Comment:
the type signature of `render` is `render(): React.Node`  (see [here](https://flow.org/en/docs/react/types/#toc-react-node)) so this should be:

```js
import * as React from 'react';

render(): React.Node {}
```